packages/circuits/package.json

```diff
@@ -33,11 +33,11 @@
     },
     "dependencies": {
         "@ethersproject/solidity": "^5.7.0",
+        "@semaphore-protocol/identity": "^3.5.0",
```

Comment:
Actually I should have noticed this before, but this should stay a dev dependency. It's only used in tests.

Reply:
be762b154d292b5ca7cb8b884c81da998b924f2c